staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -33,20 +33,18 @@ import (
 // HTTP2 clients). Pure HTTP clients should use the RoundTripper returned from
 // New.
 func HTTPWrappersForConfig(config *Config, rt http.RoundTripper) (http.RoundTripper, error) {
-	if config.WrapTransport != nil {
```

Comment:
this field is independent of the auth provider, and can apply in cases where no auth provider is specified, or a method other than token auth is used.